### ISSUE TITLE
Is having a CDIS a good idea?

### ISSUE THREAD

**Issue by AnnaFeiler** [IHE/sdpi-fhir#252]
## Section Number

Volume 1, Appendix C.1.5 Application of ISO/IEC 60601-1-8 Concepts & Definitions

## Priority

Medium

## Issue

I struggle with the idea of having a user confirm an alarm using a system without reliable distribution of alarms. Has anyone looked at the risk management implications (foreseeable misuse, unreliable confirmation mechanisms…)?

## Proposed Change

Since we only have a DIS at the time being, I’d delete this and limit the whole section to DIS topics and things we are only reproducing from -1-8 (second part of this is optional to me, but since it is already there and well written we might as well keep it.).

**Comment by ToddCooper** [comment]
We did consider this -- omitting this section -- however, these use cases are both for the 1.0 functional requirements, but also forward looking to future updates 1.x/2.0 etc.   We did omit some use cases that are totally deferred, such as isolation point-of-care.  

It is a judgement call for what is best for readers of the specification.  

Note: that use case requirements are mapped from profile back to Appendix C sections.  So in the case of those scenarios where the functional requirements are not implemented in a given version, that will be clear from the lack of mappings from profiles back to use cases.  IOW the use of requirements traceability ...
